lib/Model/Importer/DebugInfo/DwarfImporter.cpp

```diff
@@ -537,11 +502,11 @@ class DwarfToModelConverter : public BinaryImporterHelper {
     case llvm::dwarf::DW_TAG_typedef:
     case llvm::dwarf::DW_TAG_restrict_type:
     case llvm::dwarf::DW_TAG_volatile_type: {
-      model::QualifiedType TargetType = rc_recur getTypeOrVoid(Die);
-      auto *Typedef = cast<model::TypedefDefinition>(T);
-      Typedef->OriginalName() = Name;
-      Typedef->UnderlyingType() = TargetType;
-      revng_assert(Typedef->UnderlyingType().UnqualifiedType().isValid());
+      auto &Typedef = cast<model::TypedefDefinition>(*Definition);
+      Typedef.OriginalName() = Name;
+      Typedef.UnderlyingType() = rc_recur makeTypeOrVoid(Die);
+      // revng_assert(Typedef.UnderlyingType()->verify(true));
+      // TODO: decide whether we want this assert.
```

Comment:
Context?

Reply:
I don't remember. I guess it is more strict than how it was before, but I have no idea why I thought this might break something.

I'm pretty sure we should just drop the comment.

Reply:
Done.